cockroachcloud/cluster-management.md

```diff
@@ -148,6 +148,16 @@ You can add or remove up to nine regions at a time through the Console. Note tha
 1. In the **Confirmation** dialog, verify your new cluster configuration.
 1. Click **OK**.
 
+## Create a database
+
+You can use the [**Databases** page](databases-page.html) to create a new database from the {{ site.data.products.db }} Console.
+
+1. Navigate to the **Databases** page from the **Overview** page of your cluster.
+1. Click **Add database**.
+1. Enter a name for the new database.
+1. *(Optional)* Deselect the checkbox if you don't want this to be the default database used when connecting to your cluster.
```

Comment:
```suggestion
```

Reply:
There is no checkbox for selecting the default database. This is because the default database behaviour is handled more elegantly with this PR: https://github.com/cockroachlabs/managed-service/pull/8903